src/main/java/com/auth0/client/mgmt/GuardianEntity.java

```diff
@@ -23,6 +23,7 @@
 
     /**
      * Create a new Guardian Enrollment Ticket. A token with scope create:guardian_enrollment_tickets is needed.
+     * <p/>
```

Comment:
I just want a line break. The IDE didn't marked that as a warning. Any other option?